backend/src/bundles/users/user.service.ts

```diff
@@ -27,18 +28,14 @@ class UserService implements Service {
         };
     }
 
-    public async create(
-        payload: UserSignUpRequestDto,
-    ): Promise<UserSignUpResponseDto> {
-        const user = await this.userRepository.create(
+    public async create(payload: UserSignUpRequestDto): Promise<UserEntity> {
```

Comment:
Have you tried to create 2 users with a same email? I just can`t find where the error is handled

Reply:
Hello, Pavlo. For this one I decided to throw error in auth.service and have catch block in the controller.